cores/arduino/stm32/twi.c

```diff
@@ -101,7 +101,7 @@
   */
 
 /*  Family specific description for I2C */
-#define I2C_NUM (5)
+#define I2C_NUM (4)
```

Comment:
can this value be defined in the family / target specific files ?

Reply:
It could be something like this:
```
#if defined(STM32F7xx) || defined(STM32L4xx)
#define I2C_NUM (4)
#elif defined(STM32F0xx) || defined(STM32F1xx) || defined(STM32L1xx)
#define I2C_NUM (2)
#else // STM32F2xx || STM32F3xx || STM32F4xx || STM32L0xx
#define I2C_NUM (3)
#endif
```

Reply:
Done!